commands/ssh.go

```diff
@@ -1,21 +1,18 @@
 package commands
 
 import (
-	"io"
-	"os"
+	"fmt"
 	"strings"
 
 	"github.com/docker/machine/log"
 
 	"github.com/codegangsta/cli"
 	"github.com/docker/machine/drivers"
-	"github.com/docker/machine/ssh"
 )
 
 func cmdSsh(c *cli.Context) {
 	var (
-		output ssh.Output
-		err    error
+		err error
```

Comment:
I don't think you need this declaration here since err is defined many times below.
Or at least can do `var err error` instead of doing the block.